src/app/components/Statistics/CO2ConsumptionCharts/index.tsx

```diff
@@ -42,13 +44,28 @@ const DateRangeContainer = styled.div`
     }
 `
 
+interface TextProps {
+    backgroundColor: string
+}
+
+const Text = styled('p')<TextProps>`
+    margin-left: 3rem !important;
+    margin-right: 1rem !important;
+    margin-top: 0.5rem !important;
+    margin-bottom: 0.5rem !important;
```

Comment:
I don't like the use of `!important` here, is it necessary? If not, can you please remove it?

Reply:
Removed